insights/parsers/httpd_conf.py

```diff
@@ -89,39 +77,39 @@
 from .. import Parser, parser, get_active_lines, LegacyItemAccess
 
 
+SUPPORTED_SECTION_BEG = ('<IfModule', '<VirtualHost')
+SUPPORTED_SECTION_END = ('</IfModule', '</VirtualHost')
+
+
 @parser('httpd.conf', filters=['IncludeOptional'])
 @parser('httpd.conf.d')
 class HttpdConf(LegacyItemAccess, Parser):
     """
     Get the key value pairs separated on the first space, ignoring leading
-    and trailing spaces.  The "<IfModule prefork.c>" and "<IfModule worker.c>"
-    sections are parsed into 'MPM_prefork' and 'MPM_worker' sub-dictionaries
-    respectively.
+    and trailing spaces.
 
     If the file is ``httpd.conf``, it also stores first half, before
     ``IncludeOptional conf.d/*.conf`` line, and the rest, to the ``first_half``
     and ``second_half`` attributes respectively.
 
     Attributes:
-        data (dict): Dictionary of parsed data in format of {option: value} or
```

Comment:
@kgrant-rh I think he actually got rid of what was `data` before and only retains `full_data` and renamed it to `data` (as it is actually a super-set). As `master` branch can break backwards compatibility, I think it is a good move - if I understood the change correctly.